docs/core-concepts/core-concepts-overview.md

```diff
@@ -0,0 +1,199 @@
+---
+id: core-concepts
+title: Core Concepts
+sidebar_label: Core Concepts
+---
+
+COMIT is an open protocol facilitating cross-blockchain applications.
+This part of the documentation explains core concepts and how they relate to the COMIT protocol.
+The documentation contains embedded google slides for explaining details.
+Note that you can play/pause the slide-player and step back and forth. 
+
+More detailed information about the COMIT protocol can be found in [COMIT protocol stack](comit-protocol-stack.md).
+
+## Atomic Swaps using HTLCs
+
+[Atomic Swap](#atomic-swap)s using [HTLC](#htlc)s defines solution using time-locks for exchanging two digital assets without the need of a trusted third party.
+
+Prior to the actual swap execution there is usually a [Negotiation](#negotiation---maker-and-taker) phase where the two parties agree on the ledgers, assets and amounts to be swapped.
+
+Assuming, that Alice has ETH on Ethereum and Bob has BTC on Bitcoin, Alice wants Bob's BTC and Bob wants Alice's ETH, the execution of a swap would be like this:
+
+1. Alice's sets up by generating a secret and hashing it. She sends to Bob:
+    1. Hash of the secret. 
+    1. Her refund address on Ethereum.
+    1. Her redeem address on Bitcoin.
+    1. Proposed expiry times on the for the [HTLC](#htlc)s. (Note: That could also be done by Bob.)
```

Comment:
🙃 Can you introduce footnotes instead of notes in brackets?

Reply:
❓ Also, do we then assume that the expiries haven't been negotiated beforehand?

Reply:
The bracket was just removed, because this information is not especially important here. 
To answer your question: As far as I remember atm the expiries *might* be negotiated (because they can be price relevant), but at this point of the documentation we might not go into detail about that. I can mention it in the negotiation vs. Execution section (that section will change a bit in the split-up structure, https://github.com/comit-network/comit.network/pull/20)